collect_app/src/main/java/org/odk/collect/android/application/initialization/ApplicationInitializer.java

```diff
@@ -150,9 +150,10 @@ private void initializeMapFrameworks() {
     }
 
     private void importExistingProjectIfNeeded() {
-        if (!appStateProvider.isFreshInstall() && projectsRepository.getAll().isEmpty()) {
+        if (!appStateProvider.isFreshInstall() && !metaSettings.getBoolean(MetaKeys.ALREADY_TRIED_TO_IMPORT_EXISTING_PROJECT)) {
```

Comment:
I'm not sure I fully understand why this is needed? I'm guessing we end up with a problematic state if not?

Reply:
Because if you remove all projects you are moved to the `First Launch Screen`. Then a user can kill the app and open it again what would trigger importing the existing project what I want to avoid (I want that importing to be triggered just once during upgrading to the version with projects).

Reply:
Right that makes sense. Thanks. Could we change the name to be `EXISTING_PROJECT_IMPORTED`? I think the `TRIED` made me think it might be related to an earlier failure or something.

Reply:
Fixed.